File: Subject Archive/INFO1910 Intro to programming/exams/lwandm/extras/tennanachi.c
#include <stdio.h>
#include <stdlib.h>

... tennanachi(... n) {
    // TODO: implement tennanachi(n) with reasonable types
    return 0;
}

int main(int argc, char** argv) {
    if (argc != 2) {
        fprintf(stderr, "Usage: %s <n>\n", argv[0]);
        return 1;
    }
    unsigned n = strtol(argv[1], NULL, 10);
    printf("Tennanachi(%u) = %llu\n", n, tennanachi(n));
    return 0;
}

File: Subject Archive/INFO1910 Intro to programming/exams/lwandm/extras/tennanachi.py
from sys import argv, exit

def tennanachi(n):
    #TODO: return the nth tennanachi number
    return 0

if __name__=='__main__':
    if len(argv) != 2:
        print('Usage: {} <n>'.format(argv[0]))
        exit(1)
    print(tennanachi(int(argv[1])))
